pkgs/development/python-modules/wsgitools/default.nix

```diff
@@ -20,8 +21,11 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
-  # the built-in asyncore library was removed in python 3.12
-  dependencies = lib.optionals (pythonAtLeast "3.12") [ pyasyncore ];
+  # the built-in asyncore and cgi libraries were removed in python 3.13
+  dependencies = lib.optionals (pythonAtLeast "3.13") [
+    pyasyncore
+    legacy-cgi
```

Comment:
Why do we have both `standard-cgi` and `legacy-cgi`, maybe we should drop one of them.

Reply:
oh, interesting find! the [official docs](https://docs.python.org/3/library/cgitb.html) recommend [legacy-cgi](https://pypi.org/project/legacy-cgi/) but [python-deadlib](https://github.com/youknowone/python-deadlib) seems to have made its way into nixpkgs as `standard-*`. I haven't tried it, but switching to `standard-cgi` might be right "right" thing to do here

Reply:
moving everything `legacy-cgi` -> `standard-cgi` probably won't be a thing (even if the libraries might technically be compatible / do the same thing) since at least one project explicitly defines a dependency on `legacy-cgi`: https://github.com/Pylons/webob/blob/main/setup.py#L56-L58

I assume as more people start introducing "compatibility" with 3.13, that's just gonna get more and we'll end up needing both

for `wsgitools` (this PR), I think it really doesn't matter